app/uk/gov/hmrc/apiplatformmicroservice/thirdpartyapplication/connectors/ThirdPartyApplicationConnector.scala

```diff
@@ -104,18 +103,12 @@ private[thirdpartyapplication] abstract class AbstractThirdPartyApplicationConne
     http.GET[Option[Application]](s"$serviceBaseUrl/application/${applicationId.value}")
   }
 
-  def fetchApplications(developerId: DeveloperIdentifier)(implicit hc: HeaderCarrier): Future[Seq[ApplicationId]] = {
-    developerId match {
-      case EmailIdentifier(email) => http.GET[Seq[ApplicationResponse]](s"$serviceBaseUrl/application", Seq("emailAddress" -> email)).map(_.map(_.id))
-      case UuidIdentifier(userId) => http.GET[Seq[ApplicationResponse]](s"$serviceBaseUrl/developer/${userId.value}/applications").map(_.map(_.id))
-    }
+  def fetchApplications(userId: UserId)(implicit hc: HeaderCarrier): Future[Seq[ApplicationId]] = {
+    http.GET[Seq[ApplicationResponse]](s"$serviceBaseUrl/developer/${userId.value}/applications").map(_.map(_.id))
```

Comment:
should this be calling 'userId.asText' rather than .value.toString? they happen to be the same thing at the moment but why add the abstraction and not use it?

Reply:
We'll remove the asText thing